local.properties

```diff
@@ -1 +1,8 @@
-sdk.dir=C:\\AndroidSDK
\ No newline at end of file
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Tue Oct 16 13:21:29 CEST 2021
+sdk.dir=C\:\\Users\\Win7Pro64\\AppData\\Local\\Android\\Sdk
```

Comment:
This PR should not change "local.properties". Please revert the changes.

Reply:
Fixed.